lib/main.dart

```diff
@@ -1,5 +1,5 @@
 // Note: For now, Bottom Navigation Bar works on MacOS and Chrome but not iOS simulator.
-
+//majd
```

Comment:
Having the app in one file limits the scalability and also the current structure may mix various concerns (UI, logic, state management) within a single file, which could be separated into different files or layers (such as models, views, and controllers/providers) for cleaner architecture, particularly in larger applications.